docs/reference-config-guide.md

```diff
@@ -18,8 +18,8 @@ these criteria:
 ### Reference CR Groupings
 
 The most important thing included in the metadata.yaml is the list of all the CR templates that are included in the
-reference. The file includes an hierarchy of all the CR templates by parts and components. Parts are Groups of
-Components and components are groups of CR templates.
+reference. The file includes an hierarchy of all the CR templates by parts and components. parts are Groups of
```

Comment:
maybe here it should be Parts (capitalized) because its in a beginning of a sentence, but i don't know maybe its confusing

Reply:
Ah yes sorry I meant to fix put that one back and forgot.

Reply:
Fixed